src/App.test.jsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom/';
+import App from './App';
+
+describe('App', () => {
+  it('upon click, link takes user to a list of names', async () => {
+    render(
+      <MemoryRouter
+        initialEntries={['/', '/list', '/detail/1']}
+        initialIndex={0}
+      >
+        <App />
+      </MemoryRouter>
+    );
+    const listLink = screen.getByText('Character List');
+    expect(listLink).toBeInTheDocument();
```

Comment:
To satisfy the testing of detail page with initialEntries, we will want to use initialEntries={['/detail/1'}]. This will pull the id from the url and allow Rick Sanchez to be populated in the test suite. Also, don't forget to add in a behavioral test that navigates from the list to detail view.